src/update_flash.c

```diff
@@ -194,6 +194,91 @@ static int RAMFUNCTION wolfBoot_copy_sector(struct wolfBoot_image *src,
     return pos;
 }
 
+static int wolfBoot_swap_and_final_erase(int resume)
```

Comment:
Is this function always needed? Should it be wrapped with `#ifndef DISABLE_BACKUP`?

Reply:
I had thought it was never called for DISABLE_BACKUP but it's also called in wolfBoot_start, good point